editor/src/clj/editor/gl/vertex2.clj

```diff
@@ -145,113 +96,89 @@
 (definline buf-put-floats! [buffer byte-offset numbers]
   `(buffers/put-floats! ~buffer ~byte-offset ~numbers))
 
-(definline buf-put! [buf byte-offset data-type normalize numbers]
-  `(buffers/put! ~buf ~byte-offset ~data-type ~normalize ~numbers))
-
-(defn put!
-  ^VertexBuffer [^VertexBuffer vbuf byte-offset data-type normalize numbers]
-  (buf-put! (.buf vbuf) byte-offset data-type normalize numbers)
-  vbuf)
+(definline buf-put! [buf byte-offset buffer-data-type normalize numbers]
+  `(buffers/put! ~buf ~byte-offset ~buffer-data-type ~normalize ~numbers))
```

Comment:
Removed unused `put!` function and renamed `data-type` argument to `buffer-data-type` to differentiate it from the vertex attribute `data-type` that is present in many places where we call this from.